src/helpers/xml-builder.js

```diff
@@ -179,6 +179,22 @@ const buildTableRowHeight = (tableRowHeight) =>
     .att('@w', 'hRule', 'atLeast')
     .up();
 
+const buildHorizontalRule = () => {
+  const hrFragment = fragment({ namespaceAlias: { w: namespaces.w } }).ele('@w', 'p');
+  const pPr = hrFragment.ele('@w', 'pPr');
+  const pBdr = pPr.ele('@w', 'pBdr');
+  pBdr.ele('@w', 'bottom', {
```

Comment:
Small pieces of feedback - 

1. Can you see if we can grab the parameters from the style tag to make this dynamic? You can see a few styles here https://www.w3schools.com/tags/tag_hr.asp but recommend testing with what TinyMCE supports too. 

2. If there are no styles applied can we pull the defaults from the documentDefaults in the constants file?

3. Can we make a parameter (and add it to the `README.md`) to allow someone to override the defaults when instantiating the html-to-docx library?

Reply:
Last PR where I did this with the preprocessing flag although that was just doing true/false. Same idea though

https://github.com/TurboDocx/html-to-docx/pull/51/files